src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -926,5 +932,22 @@ internal int GetIdentity()
 
             return hashCode;
         }
+
+        private static string FormatNotficationText(Notification notification)
+        {
+            string message = notification.Message.Text?.Trim() ?? string.Empty;
+
+            if (!string.IsNullOrWhiteSpace(notification.Exception?.Kind?.Trim()))
```

Comment:
>notification.Exception?.Kind?.Trim() [](http://example.com/codeflow?start=43&length=36)

Can you store this in a variable so you don't have to trim twice?

Reply:
fixed